hpat/tests/gen_test_data.py

```diff
@@ -5,102 +5,132 @@
 import pandas as pd
 
 
+class ParquetGenerator:
```

Comment:
Are you sure we need a class here (same for `SparkGenerator`)? It looks like it can be just a functions. it is ok, but I think it is better to make things simpler.

Reply:
`class ParquetGenerator` stores information about the generated data. If replace methods with functions, will have to use global variables to control generated data state. Using global variables is not good and I did not think of a third option :)

The class for another generator(`class SparkGenerator`) is really not needed yet, so I replaced it with function.